executable/swf.ksy

```diff
@@ -192,14 +417,44 @@ enums:
     0x46: none # F
     0x43: zlib # C
     0x5a: lzma # Z
+  join_style:
+    0: round_join
+    1: bevel_join
+    2: miter_join
+  cap_type:
+    0: round_cap
+    1: no_cap
+    2: square_cap
+  fill_style_type_enum:
+    0x00: solid
+    0x10: linear_gradient
+    0x12: radial_gradient
+    0x13: focal_gradient
+    0x40: repeat_bitmap
+    0x41: clipped_bitmap
+    0x42: non_smoothed_repeat_bitmap
+    0x43: non_smoothed_clipped_bitmap
+  spread_mode_enum:
+    0: pad_mode
+    1: reflect_mode
+    2: repeat_mode
+    3: reserved
+  interpolation_mode_enum:
+    0: normal_rgb
+    1: linear_rgb
+    2: reserved
```

Comment:
won't the same ids cause issues?

Reply:
> won't the same ids cause issues?

Yes, they certainly do. I don't know a target language where this works as intended - usually a parse error is thrown (C++, C#, Go, Java, Nim, Python, PHP) or the last value shadows all previous ones (silently in JavaScript, Lua, Ruby; in Perl it shows a warning).